transports/sse/integration-tests/src/test/java/io/quarkiverse/mcp/server/sse/it/ServerFeaturesTest.java

```diff
@@ -115,16 +122,23 @@ public void testResource() throws URISyntaxException {
                 "file:///project/alpha");
     }
 
-    protected URI initClient() throws URISyntaxException {
+    protected static McpSseClient createMcpSseClient(URI testUri) {
+        String testUriStr = testUri.toString();
+        if (testUriStr.endsWith("/")) {
+            testUriStr = testUriStr.substring(0, testUriStr.length() - 1);
+        }
+        return new McpSseClient(URI.create(testUriStr + "/mcp/sse"));
+    }
+
+    protected URI subscribeClient(URI testUri) {
```

Comment:
This should still be called `initClient()` because it performs the "initialization" phase (as described in the spec).